tools/downloader/src/open_model_zoo/model_tools/_common.py

```diff
@@ -110,3 +128,44 @@ def get_package_path(python_executable, package_name):
     # For a package, the file is __init__.py, so to get the package path,
     # take its parent directory.
     return file_path.parent
+
+def get_version():
+    get_ie_version = subprocess.run(
+        [str(sys.executable), '-c',
+            'from openvino.inference_engine import ie_api;'
+                'print(ie_api.get_version())',
+        ],
```

Comment:
We shouldn't be trying to get the IE version. OMZ is not IE, and IE might not even be in `PYTHONPATH`; furthermore, loading IE would add extra overhead. We should instead use our own `version.txt` file. Make sure to only try to access the file if we are not installed inside a Python environment (IOW, if `_OMZ_ROOT` is set).

Reply:
Done